### DISCUSSION TITLE
How To Setup Chia Coin Harvester To Farm Chia On Multiple Computers on ubuntu?

### DISCUSSION THREAD

**Discussion by mohsintirmizi** [Chia-Network/chia-blockchain#7878] [General]
I saw someone doing this windows version on youtube, but i need to know if I can do it on ubuntu too. 
How To Setup Chia Coin Harvester To Farm Chia On Multiple Computers on ubuntu?
reason is this that i created the plots in ext4 format, and now cant format all the drives and re-plot :D

**Comment by rmica** [comment]
Setup an ubuntu box. Install chia-blockchain and run "chia init" ; copy .chia/mainnet/config/ssl/ca directory from the farmer node to a temporary dir on the new harvester, say ~/farmca ; run "chia init -c ~/farmca" ; edit ~/.chia/mainnet/config/config.yaml , go to harvester section and set farmer_peer/host with IP address or FQDN of your farmer node ; add your plot drives mountpoints to plot_directories ; (highly suggested) go to logging section and set log_level to "INFO" ; run "chia start harvester" and follow the log file. You should see a successfull connection with the farmer followed by messages showing new challenges received and their filter/proof status.
Hope this helps.

**Comment by keliew** [comment]
https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines
